src/kyc.ts

```diff
@@ -35,21 +36,32 @@ export class KycApi {
    * perform KYC, or KYB, on the user or business based upon which token is
    * populated: userToken or businessToken.
    */
-  async verify(source: {
+  async verify(type: string, token: string, options: {
     notes?: string,
-    userToken?: string,
-    businessToken?: string,
     manualOverride?: boolean,
     referenceId?: string,
   } = {}): Promise<{
     success: boolean,
     kyc?: Kyc,
     error?: MarqetaError,
   }> {
+    if (!type) {
+      return {
+        success: false,
+        error: mkError('This call requires a verification "type" parameter of ' +
+          '"business" or "user".'),
```

Comment:
Why not make this one line? Yes, it's longer, but it's understandable in this usage...